charts/acp/values.yaml

```diff
@@ -81,10 +81,104 @@ ingress:
 
   ## Ingress TLS configuration
   ## Secrets must be manually created in the namespace
+  ## or automatically using `tlsSecrets` variable
   tls: []
-  #  - secretName: acp-tls
-  #    hosts:
-  #      - acp.domain.com
+  # - secretName: ingress-tls
+  #   hosts:
+  #     - acp.acp-system
+
+  ## Ingress TLS secrets
+  ## List of certificates to be created for Ingress
+  tlsSecrets: []
+  # - name: ingress-tls
+  #   cert: |
+  #     -----BEGIN CERTIFICATE-----
```

Comment:
I would rather not put example the whole cert and key here, as this makes values.yaml really long

Reply:
I removed the content of the example cert.
I kept `-----BEGIN CERTIFICATE-----` -like strings to indicate that both the certificate and key must be in PEM format.